### ISSUE TITLE
Simple conversion mode

### ISSUE THREAD

**Issue by golaz** [E3SM-Project/e3sm_to_cmip#50]
e3sm_to_cmip is a very useful utility. 

Beyond generating data for CMIP release, e3sm_to_cmip could also be very useful for E3SM internal use. For example, when we need to convert model output variables to CMIP naming convention in order to use diagnostics package that follow CMIP variable naming convention.

Right now, e3sm_to_cmip can be used to do that, but it's a cumbersome because of all the CMIP6/CMOR "stuff". 

I'd like to suggest adding a simple conversion more of operation where e3sm_to_cmip only converts/renames variables without any extra CMIP stuff (CV, filename convention, output directory structure, etc...).

Basically go from a set of input time-series files generated by NCO like

```
CLDTOT_187001_201412.nc
SOLIN_187001_201412.nc
```

to something like:

```
clt_187001_201412.nc
rsdt_187001_201412.nc
```

using simply

`e3sm_to_cmip --convert -v clt, rsdt --input <input_path> --output <output_path>`

In this case, only variables and their metadata would change. Everything else would remain unchanged. This should also work regardless of the input file time frequency.

**Comment by sterlingbaldwin** [comment]
How much of the metadata should remain intact? For example, would you just want the raw data, or would you also want things like the standard name?

For example, here's the full CMIP6 variable info for clt:
```
        float clt(time, lat, lon) ;
                clt:standard_name = "cloud_area_fraction" ;
                clt:long_name = "Total Cloud Cover Percentage" ;
                clt:comment = "Total cloud area fraction (reported as a percentage) for the whole atmospheric column, as seen from the surface or the top of the atmosphere. Includes both large-scale and convective cloud." ;
                clt:units = "%" ;
                clt:cell_methods = "area: time: mean" ;
                clt:cell_measures = "area: areacella" ;
                clt:missing_value = 1.e+20f ;
                clt:_FillValue = 1.e+20f ;
```
Which, if any, of these metadata fields should be retained? I have a quick-and-dirty method of doing the simple conversion nearly working, but it doesnt retain any of the variable metadata.

**Comment by golaz** [comment]
@sterlingbaldwin:  I don't think we need to retain any of the original **variables attributes**. Those can be safely replaced wit the CMIP variable metadata. 

For the **global attributes**, keep all original metadata intact and no need to add any CMIP stuff. Maybe just add a history.

That may already what you have implemented?

**Comment by mccoy20** [comment]
Just to add to that, I think @golaz 's comment was on global attributes but @sterlingbaldwin may have been asking about the variable's attributes. I would advocate that the variable itself should have all the attributes that the CMIP6 variable would get, so as in @sterlingbaldwin 's example the new produced 'clt' should have all the attributes that are listed in the example. I think Chris says not to worry about retaining old variable's names and any attributes it had, and I agree with that. The global attributes are different and it may be nice to have an ability to define and add a global attribute, but I agree that the history attribute that lists the command that was used to convert the variable would be enough as a minimum requirement.
